chairman/configuration/defaults/simpleview/config-0.yaml

```diff
@@ -0,0 +1,212 @@
+# global filter; messages must have at least this severity to pass:
```

Comment:
@newhoggy We should put these in the `configuration` folder. Previously we had configurations all over the place. It just makes it easier in the events where we want to add a tracer for example, they all exist in `configuration`.